src/docs/framework-integration/angular.md

```diff
@@ -9,82 +9,228 @@ contributors:
+
+```bash
+ng new angular-workspace --no-create-application
+cd angular-workspace
+ng generate library component-library
+```
+
+### Create a Stencil Component Library
+
+Your Stencil library will be the location that you write your web components. 
+
+```bash
+npm init stencil components stencil-library
+```
+
+### Install the Angular Output Target in your Stencil Component Library
+
+Now that the project structure is set up, we can install the Angular Output Target package. This package contains the Angular wrapper function that we will use to generate our Angular wrapped components. To install the Angular Output Target package, run the following command in your Stencil project directory
+
+`npm i @stencil/angular-output-target`
+
+or
+
+`yarn add @stencil/angular-output-target`
+
+### Add the Angular Wrapper Function to your Stencil Component Library
+With the Angular Output Target package installed, we can now configure our Stencil Component Library to build our Angular wrapped components. In the `stencil.config.ts` file of your Stencil component library, add the Angular wrapper function. 
+
+Within your Stencil’s config file, you can import the Angular wrapper function for use within the outputTargets array. If you copy and paste this, ensure you update the “my-workspace” and “my-lib” to coincide with what you named your component library, as well as to update the `componentCorePackage` to the name of your Stencil component library.
 
 ```tsx
-import { BrowserModule } from '@angular/platform-browser';
-import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
+import { angularOutputTarget as angular } from '@stencil/angular-output-target';
+export const config: Config = {
+  namespace: 'demo',
+  outputTargets: [
+    angular({
+      componentCorePackage: `your-stencil-library-name`,
+      directivesProxyFile: `../your-angular-workspace-name/projects/your-angular-library-name/src/lib/stencil-generated/components.ts`
+    }),
+    {
+      type: 'dist',
+      esmLoaderPath: '../loader',
+    },
+    {
+      type: 'dist-custom-elements',
+    },
+  ],
+};
+```
 
-import { AppComponent } from './app.component';
+Once you have added this and have `directivesProxyFile` pointing to the correct directory of your Angular component library, you can build Stencil. 
 
-@NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, FormsModule],
-  bootstrap: [AppComponent],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
-})
-export class AppModule {}
+`npm run build`
+
+Or
+
+`yarn build`
+
+Once the build is complete, you will see new files in your Angular component library's directory!
+
+
+```
+top-most-directory/
+├── stencil-library/
+│   ├── stencil.config.js
+│   └── src/components
+└── angular-workspace/
+    └── projects/
+        └── component-library/
+            └── src/
+                ├── lib/
+                │   └── **stencil-generated/components.ts**
```

Comment:
Ugh. Looks like the bold didn't work here. Wonder if there's something else we can do. Maybe a comment?

Reply:
Yeah, the code processing will likely not allow us to do that... and a comment may stick out too much... Ummmmmm... Let's actually maybe remove this flourish until we can figure out a better solution in the future.